server/src/main/java/io/druid/server/lookup/cache/LookupCoordinatorManagerConfig.java

```diff
@@ -26,63 +26,45 @@
 
 public class LookupCoordinatorManagerConfig
 {
-  public static final Duration DEFAULT_HOST_DELETE_TIMEOUT = Duration.millis(1_000L);
-  public static final Duration DEFAULT_HOST_UPDATE_TIMEOUT = Duration.millis(10_000L);
-  public static final Duration DEFAULT_DELETE_ALL_TIMEOUT = Duration.millis(10_000L);
-  public static final Duration DEFAULT_UPDATE_ALL_TIMEOUT = Duration.millis(60_000L);
-  @JsonProperty
-  private Duration hostDeleteTimeout = null;
-  @JsonProperty
-  private Duration hostUpdateTimeout = null;
+  public static final Duration DEFAULT_HOST_TIMEOUT = Duration.millis(2_000L);
+  public static final Duration DEFAULT_ALL_HOST_TIMEOUT = Duration.millis(900_000L);
```

Comment:
this is the timeout used for waiting on allFuture to lookup management to complete on *all* nodes and not just in one tier as was done in older code , so increased.